.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Purpose of this PR
+
+* Description of feature/change.
+* Links to screenshots, design docs, user docs, etc.
```

Comment:
We definitely want to mention that they should include related issues here.

Reply:
We also could add which part of OpenTK this affects (Math, Core, GL, platform, etc)